common/components/BalanceSidebar/EquivalentValues.tsx

```diff
@@ -33,9 +34,13 @@ export default class EquivalentValues extends React.Component<Props, {}> {
                     </span>
                     <span className="EquivalentValues-values-currency-value">
                       {' '}
-                      {balance
-                        ? formatNumber(balance.amount.times(rates[key]))
-                        : '???'}
+                      {balance ? (
+                        balance.isPending ? (
+                          <Spinner size="x1" />
+                        ) : (
+                          formatNumber(balance.amount.times(rates[key]))
+                        )
+                      ) : null}
```

Comment:
Can be simplified to `{balance && onlyRenderMeIfBalanceIsTruthy}`. The only caveat with that method is that if `balance` were an integer, you would do `!!balance` instead.